hooks/pre-commit.sample

```diff
@@ -3,15 +3,14 @@
 set -e
 # format checks
 cargo fmt --all -- --check
-cd ./wasm_samples/default_executor
-cargo fmt --all -- --check
-cd -
 cd ./wasm_samples
 cargo fmt --all -- --check
 cd -
 # update the default executor
 cargo run --release --bin iroha_wasm_builder -- build ./wasm_samples/default_executor --optimize --out-file ./defaults/executor.wasm
 # update the default genesis, assuming the transaction authority is `iroha_test_samples::SAMPLE_GENESIS_ACCOUNT_ID`
+mkdir -p defaults/blobs
+rm defaults/blobs/*
 cargo run --release --bin kagami -- genesis generate --executor-path-in-genesis ./executor.wasm --genesis-public-key ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 > ./defaults/genesis.json
```

Comment:
After this PR merge, you will locally need to re-generate `genesis.json` as needed to get correct wasm blobs referenced as hashes in `genesis.json`

Reply:
When [approach changed](https://github.com/hyperledger/iroha/pull/5027#discussion_r1787878806), you will be running another command introduced by #5123 to get wasm blobs